pandas_datareader/edgar.py

```diff
@@ -20,6 +20,8 @@
 _SEC_FTP = 'ftp.sec.gov'
 
 _COLUMNS = ['cik', 'company_name', 'form_type', 'date_filed', 'filename']
```

Comment:
I do. `_COLUMNS` and `_COLUMN_TYPE` look similar and are used in `read_csv()`, but the first is used for naming columns and the second for `dtype`. I need to name the columns because the cleanest way of removing the varying lines of metadata (across small format changes over time) at the top of each csv also removes the header row.